mod-data-import-converter-storage-server/src/main/java/org/folio/rest/impl/DataImportProfilesImpl.java

```diff
@@ -737,6 +740,12 @@ public void deleteDataImportProfilesActionProfilesById(String id, String lang, M
     });
   }
 
+  private boolean isDeletedProfile(String id, String oclcCreateInstanceActionProfileId,
```

Comment:
@oleksandr-dekin  I think this is not a valid method-name for current logic. We don't check if they were deleted, we check if these profiles can be deleted/updated(if they are not immutable).

Reply:
@VRohach make sense, I rename method and refactored all logic to verify uuids in this method, please double-check